sys-apps/systemd/systemd-9999.ebuild

```diff
@@ -558,11 +561,19 @@ pkg_postinst() {
 	# Flatcar: Reenabling systemd-timesyncd service too.
 	systemd_reenable systemd-networkd.service systemd-resolved.service systemd-timesyncd.service
 
+	# Flatcar: TODO: Possibly replace `systemctl enable` with
+	# `systemd_enable_service`, so it potentially could enable the
+	# services by making symlinks in /lib, instead of /etc?
+	# Currently does nothing, because ENABLED_UNITS end up being
+	# empty, but might not be true in future.
 	if [[ ${ENABLED_UNITS[@]} ]]; then
 		systemctl --root="${ROOT:-/}" enable "${ENABLED_UNITS[@]}"
 	fi
 
-	if [[ -z ${REPLACING_VERSIONS} ]]; then
+	# Flatcar: We enable getty and remote-fs targets ourselves
+	# above. The code below would modify /etc, which we don't
+	# want.
+	if false && [[ -z ${REPLACING_VERSIONS} ]]; then
```

Comment:
Can we also remove this whole `if` block?

Reply:
I can. My change effectively does that, without removing it. :)

I was just wondering what to aim at: at the minimal divergence from upstream ebuild (change one line) or for the lowest cognitive overload here (drop superfluous stuff). But since we already dropped some lines, I can drop this one too then.

Reply:
Deleted stuff is quite hard to handle when diffing new versions. Either comment it out or do what you did here, but just deleting lines can be very confusing, because then you don't know whether those lines are new or they should stay deleted.

Reply:
Dropped.

Reply:
@marga-kinvolk: I did not see your comment when I wrote it, sorry about that. Frankly speaking, commenting the lines out somehow didn't come to my mind…